README.md

```diff
@@ -18,7 +18,7 @@ And then transposed using `scripts/transpose-format.js`.
 
 ## Projects Using This API
 
-- [Pocket Decks Top](https://pocketdecks.top/) A website showing a tier list of the best Pokemon TCG Pocket Decks based on tournaments resuts, updated weekly every Monday
+- [Pocket Decks Top](https://pocketdecks.top/) A website showing a tier list of the best Pokemon TCG Pocket Decks based on tournaments results, updated weekly every Monday
```

Comment:
probably want to say "tournament results" with no `s`.

Reply:
Good spotting! Will do this in a separate PR quickly now 😃 Thanks!